lib/identity_doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -95,7 +95,17 @@ def pii_from_doc
           end
 
           pii[:state_id_type] = 'drivers_license'
+
           pii[:dob] = "#{pii[:dob_month]}/#{pii[:dob_day]}/#{pii[:dob_year]}"
```

Comment:
Turns out this approach leaves in extra fields `dob_month` `dob_day` dob_year` that are not present from the Acuant client. For consistency, any objection if I use `.delete` like I do below to keep cruft out of the resulting payload?

Reply:
Yeah, I think that's good and those things aren't used but we'll need to pay attention during TrueID testing prior to turning it on.